### DISCUSSION TITLE
Bookmarklet to dns

### DISCUSSION THREAD

**Discussion by YeeticusPrimed** [3kh0/ext-remover#668] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm trying to put a bookmarklet to a DNS. I'm using adguard dns for this. Should I use a dns rewrite?

**Comment by AshtonDavies** [comment, marked as answer]
I have tried make a DNS bookmarklet with NGINX like what the incognito one used, but it's painful to do on Windows.
